pkgs/build-support/buildenv/builder.pl

```diff
@@ -111,7 +111,11 @@ sub findFiles {
 
     unless (-d $target && ($oldTarget eq "" || -d $oldTarget)) {
         if ($ignoreCollisions) {
-            warn "collision between `$target' and `$oldTarget'\n" if $ignoreCollisions == 1;
+            if (checkCollision($oldTarget, $target)) {
+                print STDERR "collision between identical paths `$target' and `$oldTarget'\n";
```

Comment:
```
"collision between paths $target' and `$oldTarget with identical content\n"
```

is more precise, since the paths are not identical itself.

Reply:
Okay, done.